openml-h2o/pom.xml

```diff
@@ -156,9 +159,23 @@
             <scope>test</scope>
             <type>test-jar</type>
         </dependency>
+        <dependency>
+            <groupId>com.google.auto.service</groupId>
+            <artifactId>auto-service</artifactId>
+        </dependency>
```

Comment:
maybe move auto-service up so that we have all test dependencies together (openml-utils test-jar is a test dependency and is above this)

Reply:
Also, to increase visibility we can add a comment in between runtime and test dependencies